packages/app/src/domain/vaccine/vaccine-page-introduction.tsx

```diff
@@ -125,21 +139,59 @@ export function VaccinePageIntroduction({
         </Box>
       </Tile>
 
-      <Box px={{ _: 3, sm: 4 }}>
-        <ContentHeader
-          subtitle={text.description}
-          reference={text.reference}
-          metadata={{
-            datumsText: text.datums,
-            dateOrRange: data.vaccine_administered_total.last_value.date_unix,
-            dateOfInsertionUnix:
-              data.vaccine_administered_total.last_value.date_of_insertion_unix,
-            dataSources: [],
-            moreInformationLabel: text.more_information.label,
-            moreInformationLink: text.more_information.link,
-          }}
-        />
-      </Box>
+      <TwoKpiSection>
+        <Box>
+          <Heading level={3}>{pageInfo.title}</Heading>
+          <Tile>
+            <RichContent blocks={pageInfo.description} />
+            <Box spacing={{ _: 3, md: 0 }} display="flex">
```

Comment:
Don't think the spacing prop does anything with only a single child
```suggestion
            <Box display="flex">
```